models/workitemtype_test.go

```diff
@@ -71,6 +74,8 @@ func TestMarshalEnumType(t *testing.T) {
 }
 
 func TestMarshalFieldDef(t *testing.T) {
+	resource.Require(t, resource.UnitTest)
```

Comment:
Yeah.. hmm.. 

My initial thought was just that it sounded strange to 'require resource UnitTest', but if we see it more as a direct mapping to an ENV var, then it makes more sense to call it UnitTest.